src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  margin: 0;
+  display: flex;
+}
+
+.star_box {
```

Comment:
If you know about BEM methodology here is better to name an element as `.star__box`